packages/constants/package.json

```diff
@@ -1,6 +1,46 @@
 {
   "name": "@element-plus/constants",
   "private": true,
+  "homepage": "https://github.com/element-plus/element-plus#readme",
```

Comment:
It seems there is still a version missing.

Reply:
Did you mean the "version" field? Well, that field will be added once everything is ready, so to keep the process intact I don't want to add those fields related to the publishing stuff.

Reply:
Yes. OK, got it.

Reply:
We can create a release script to update all version later.